### DISCUSSION TITLE
Regarding setting up the project

### DISCUSSION THREAD

**Discussion by Ksathwik03** [avinashkranjan/Recess#261] [Q&A] [answered]
Hey guys hope all are doing good, I am sathwik from iiita can anyone help me setting up the project and saying all the requirements i need to start contributing thank you.

**Comment by avinashkranjan** [comment, marked as answer]
It's basically a React Project with Firebase as Backend,...You just need to clone the project on you local machine and run `npm i && npm start` that's basically it.
